src/services/user.service.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const Users = require('../db/models/users');
+
+function normalizeUser({ id, name }) {
+  return {
+    id,
+    name,
+  };
+}
+
+const getAll = async() => {
```

Comment:
Consider returning the list of normalized users by calling 'normalizeUser' function for each user.